src/cli.js

```diff
@@ -34,27 +34,33 @@ const cli = meow([`
   }
 }])
 
-const token = cli.flags.t || process.env.GITHUB_TOKEN
+const user = cli.flags.u || cli.flags.user
```

Comment:
This is the point of the alias section above. I think we're doing something wrong if this is needed, too?

Reply:
You are correct. Unfortunately meow isn't clearly documented (the examples don't match the reference) and reading the source code, the alias is never referred to.

In fact the library doesn't appear to provide us with anything except reading `process.argv` and a gif of a kitten puking rainbows. Undoubtedly cute, but not very useful.

So short of writing our own I think this is the best we get.

Reply:
Can we just use minimist instead, then?

Reply:
Looking at meow, it should get minimist opts fine. https://www.npmjs.com/package/minimist